proto/transcribee_proto/document.py

```diff
@@ -3,7 +3,7 @@
 from pydantic import BaseModel
 
 
-class TranscriptSegment(BaseModel):
+class Token(BaseModel):
```

Comment:
I much prefer the TranscriptSegment name because these dont nescessarily have to correspond to tokens once transcript correction happened

Reply:
maybe something like shrapnel, part, atom, bit slice, piece, hunk?

Reply:
Yeah, `Atom` works for me.